app/src/main/java/ch/sdp/vibester/activity/DownloadActivity.kt

```diff
@@ -21,16 +21,23 @@ import ch.sdp.vibester.api.ItunesMusicApi
 import ch.sdp.vibester.model.Song
 import okhttp3.OkHttpClient
 import org.w3c.dom.Text
+import java.io.File
 import java.lang.IllegalArgumentException
 /*
  * Activity that handles downloading of song extracts.
  */
 class DownloadActivity : AppCompatActivity() {
+    //Companion object to indicate when the download completes.
+    companion object {
+        var downloadComplete = false
+    }
+
```

Comment:
Why using a companion object ?

Reply:
To make it static, so that it's readable without an instance of the class.